### PR TITLE
Removed "A branch name of Sprint2 and sprint2 are just fine." #7825

### LINKED ISSUE #7825: Feedback on page: /cloud/reference/git-integration.html -- confusing sentence: "A branch name of Sprint2 and sprint2 are just fine."
# General issue

While explaining the case-insensitive unique naming of branches, the documentation goes on claming: "A branch name of `Sprint2 ` and `sprint2 ` are just fine."

## Description:

> Branch names must be case-insensitively unique. In other words, the names must be entirely unique regardless of the case you use. For example, if you have a branch named Sprint, you cannot create another branch named sprint. **A branch name of Sprint2 and sprint2 are just fine.**

This paragraph explains that the naming of branches must be unique in a case-insensitive way (i.e. you can't have two branches named Sprint and sprint). The last line is confusing the reader.

a) Are Sprint2 and sprint2 valid to be named as two different branches in the same project?
b) Are Sprint2 **or** sprint2 valid to be a branch name, either first or latter, but not both? 
c) something else?

## Possible solutions:

I'd recommend to simply remove the sentence "A branch name of Sprint2 and sprint2 are just fine", as it makes more sense without it. But not before consulting with other more experienced devdocs members first, as to what this sentence purpose is.

## Additional information:

N/A

### PR DESCRIPTION
**Pull request by medigeek**
Fixes #7825 

## Purpose of this pull request

Clarified instructions for branch naming convention.

## Affected DevDocs pages

-  https://devdocs.magento.com/cloud/reference/git-integration.html

### CODE REVIEW

**Comment by devops-devdocs** [comment]
An admin must run tests on this PR before it can be merged.

**Review by rogyar — APPROVED** [review]
Hi @medigeek. Thank you for your collaboration.

Another option might be to replace "and" with "or" and have the following sentence as a result

> A branch name of `Sprint2` or `sprint2` are just fine

But I agree that this sentence with example is more confusing than useful

File: src/cloud/reference/git-integration.md [hunk 1]
```diff
@@ -27,7 +27,7 @@ Not everyone remembers [Git](https://git-scm.com/docs) commands with ease. If yo
 In addition to Git's requirements for [valid branch names](https://www.kernel.org/pub/software/scm/git/docs/git-check-ref-format.html), {{site.data.var.ece}} adds two additional requirements:
 
 *  The `/` character isn't allowed in a branch name.
-*  Branch names must be case-insensitively unique. In other words, the names must be entirely unique regardless of the case you use. For example, if you have a branch named `Sprint`, you cannot create another branch named `sprint`. A branch name of `Sprint2` and `sprint2` are just fine.
+*  Branch names must be case-insensitively unique. In other words, the names must be entirely unique regardless of the case you use. For example, if you have a branch named `Sprint`, you cannot create another branch named `sprint`.
```
Comment (meker12) [inline src/cloud/reference/git-integration.md:30]:
```suggestion
*  Branch names are not case-sensitive. 
Each branch name must be unique regardless of the case used for each letter. For example, if you have a branch named `Sprint`, you cannot create another branch named `sprint`.
```

**Review by meker12 — CHANGES_REQUESTED** [review]

**Review by meker12 — APPROVED** [review]

**Comment by meker12** [comment]
running tests

**Pull request merged**